api_v3/lib/types/serverNode/base/KalturaServerNode.php

```diff
@@ -203,7 +203,8 @@ public function doFromObject($source_object, KalturaDetachedResponseProfile $res
 	{
 		parent::doFromObject($source_object, $responseProfile);
 		
-		if($this->shouldGet('status', $responseProfile) && !is_null($source_object->getHeartbeatTime()) && $source_object->getHeartbeatTime() < (time() - 90) && $this->status !== ServerNodeStatus::DISABLED)
+		if($this->shouldGet('status', $responseProfile) && !is_null($source_object->getHeartbeatTime())
+				&& $source_object->getHeartbeatTime(null) < (time() - 120) && $this->status !== ServerNodeStatus::DISABLED)
```

Comment:
Please make the time interval configurable, on the server-node and in the kConf.